src/components/media/MediaMenu.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as React from 'react';
+import classnames from 'classnames';
+import IconEllipsis from '../../icons/general/IconEllipsis';
+import PlainButton from '../plain-button';
+import DropdownMenu from '../dropdown-menu';
+import { Menu } from '../menu';
+import { bdlGray62 } from '../../styles/variables';
+import './Media.scss';
+
+type Props = {
+    /** Child elements */
+    children: React.Node,
+    /** Additional class names */
+    className?: string,
+    /** is the dropdown menu button disabled */
+    isDisabled: boolean,
+    /** aria-label prop for button icon */
+    label?: string,
+};
+
+const MediaMenu = ({ className, children, isDisabled, label, ...rest }: Props) => (
+    <div className={classnames('bdl-Media-menu', className)} {...rest}>
+        <DropdownMenu constrainToScrollParent isRightAligned>
+            <PlainButton isDisabled={isDisabled} type="button" aria-label={label}>
```

Comment:
It looks like this currently doesn't support adding className or resin attributes to the button/menu, which we use in VersionsSidebar, for example.